enumcover.go

```diff
@@ -160,3 +174,22 @@ func allConstsWithType(pass *analysis.Pass, targetType string) []constVal {
 	})
 	return consts
 }
+
+func buildConstMap(pass *analysis.Pass, targetType string) map[string]string {
+	initializeAllPkgs(pass)
+	constMap := map[string]string{}
+	allPkgs.Range(func(pkgKey, _ interface{}) bool {
```

Comment:
I am wondering if it's okay to re-use this sync.Map, or create a new one for this.